KsApi/mutations/inputs/EmptyInput.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public struct EmptyInput: GraphMutationInput {
+
+  public init() {}
```

Comment:
Is this initializer necessary?

Reply:
Yeah, otherwise the compiler will interpret the initializer as `internal` and the ViewModel won't be able to access it.